distributed-commandbus-jgroups/src/test/resources/org/axonframework/jgroups/commandhandling/tcp_gossip.xml

```diff
@@ -1,79 +1,24 @@
-<!--
-  ~ Copyright (c) 2010-2014. Axon Framework
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~     http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+
 
 <!--
-    TCP based stack, with flow control and message bundling. This is usually used when IP
```

Comment:
I think it can still be beneficial to have this description in here, to denote what this config is for.